srcpkgs/reuse/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'reuse'
+pkgname=reuse
+version=0.13.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-setuptools_scm gettext"
+depends="python3-binaryornot python3-boolean.py python3-debian python3-Jinja2
+ python3-license-expression python3-requests"
+checkdepends="python3-pytest $depends"
+short_desc="Helper tool for compliance with REUSE Specification"
+maintainer="k4leg <python.bogdan@gmail.com>"
+license="GPL-3.0-or-later, CC-BY-SA-4.0, CC0-1.0, Apache-2.0"
+homepage="https://pypi.org/project/reuse"
+distfiles="${PYPI_SITE}/r/reuse/reuse-${version}.tar.gz"
+checksum=89c87bd49308c8059cc36f8116872d73dc6e5788a29af8869be6532202bff801
+make_check=ci-skip  # The local build passes the tests but not the CI build.
```

Comment:
This is a `a=b; // assign b to a` style comment. Could you remove `make_check=ci-skip` for a while to show what's wrong ?

Reply:
So this is about three cases that assume no root privs. Disable them like in diffoscope template.

Reply:
Thanks!